lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -1,7 +1,15 @@
 """
 Discussion API test utilities
 """
+from contextlib import closing
+import hashlib
 import json
+from PIL import Image
+from datetime import datetime
+from pytz import UTC
+from openedx.core.djangoapps.profile_images.images import create_profile_images
+from openedx.core.djangoapps.profile_images.tests.helpers import make_image_file
+from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_names, set_has_profile_image
```

Comment:
We need to order these according to this priority http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order. Also, we try to order imports alphabetically in each category mentioned in the docs.